samples/client/petstore/php/SwaggerClient-php/phpunit.xml.dist

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit bootstrap="./vendor/autoload.php"
+         colors="true"
+         convertErrorsToExceptions="true"
+         convertNoticesToExceptions="true"
+         convertWarningsToExceptions="true"
+         stopOnFailure="false">
+    <testsuites>
+        <testsuite>
+            <directory>SwaggerClient-php/test/Api</directory>
```

Comment:
@arnested seems alright now, I've tested it with all kinds of permutations, should work fine.